### PR TITLE
Add BibTeX entry suggestion to docs.

### PR DESCRIPTION
**Pull request by lszekeres**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -116,3 +116,18 @@ a new benchmark.
...
 Please [file an issue on GitHub](https://github.com/google/fuzzbench/issues/new)
 or send a pull request fixing the problem.
+
+## How can I cite FuzzBench in my paper?
+
+You can use the following BibTeX entry:
+{% raw %}
+```
+@misc{fuzzbench,
+    author       = {Google},
```
Comment (inferno-chromium) [inline docs/faq.md:126]: As discussed on chat, lets use an authors list here. You and @jonathanmetzman can work together on an ordered list of core contributors. No rush, whenever you get a chance.

Comment (inferno-chromium) [reply docs/faq.md:126]: ping!

Comment (lszekeres) [reply docs/faq.md]: Added two options: blog post or repo -- let me know what your preferences are.

**Review by inferno-chromium — APPROVED** [review]

File: docs/faq.md (hunk 1 above)
Comment (lszekeres) [reply docs/faq.md]: I thought we'd pick one of these options

Comment (inferno-chromium) [reply docs/faq.md]: ah, lets do the github one. sorry shouldn't have committed in haste, can you fix this in master, or post a new pr.

**Pull request merged**